c2cgeoform/static/src/interactions.js

```diff
@@ -9,28 +10,35 @@ const format = new GeoJSON()
 
 export function addInteractions(options) {
   const { input, multi, ...interactionOptions } = options
+  const store = geometry => {
+    input.value = geometry ? getGeometryJSON(geometry, multi) : ''
+  }
   const draw = addDrawInteraction(interactionOptions)
   const modify = addModifyInteraction(interactionOptions)
   modify.setActive(false)
-  options.source.on(
-    'change',
-    e =>
-      (input.value =
-        options.source.getFeatures().length > 0
-          ? getGeometryJSON(
-              options.source.getFeatures()[0].getGeometry(),
-              multi
-            )
-          : '')
+  options.source.on('change', () =>
+    store(options.source.getFeatures()[0]?.getGeometry())
   )
+  // Mobile drawing
+  if (
+    window.matchMedia('(max-width: 576px)').matches &&
```

Comment:
This does not seems to be a standard breakpoint see : https://getbootstrap.com/docs/3.3/css/#grid-options

Reply:
Oops, I checked BS4 breakpoints… I'll fix it.